### PR TITLE
Add pyrepl recipe

### PR DESCRIPTION
**Pull request by kynan**

### CODE REVIEW

**Comment by kynan** [comment]
Good to go @conda-forge/core ?

**Comment by scopatz** [comment]
Thanks!

File: recipes/pyrepl/meta.yaml [hunk 1]
```diff
@@ -0,0 +1,57 @@
...
+  script: python setup.py install --single-version-externally-managed --record record.txt
+  skip: true  # [win]
+
+requirements:
+  build:
+    - python
+    - setuptools
+  run:
+    - python
+    - setuptools
```
Comment (jakirkham) [inline recipes/pyrepl/meta.yaml]: Is `setuptools` needed at runtime? If not, would recommend dropping it and rebuilding the package with a new build number.

Comment (kynan) [inline recipes/pyrepl/meta.yaml]: Good catch, probably not!

Comment (kynan) [inline recipes/pyrepl/meta.yaml]: pyrepl-feedstock#1

**Pull request merged**